wp-content/themes/vanguard-history/functions.php

```diff
@@ -624,3 +624,136 @@ function vanguard_history_all_media_for_year_story() {
 
 add_action( 'wp_ajax_vanguard_history_all_media_for_year_story', 'vanguard_history_all_media_for_year_story' );
 add_action( 'wp_ajax_nopriv_vanguard_history_all_media_for_year_story', 'vanguard_history_all_media_for_year_story' );
+
+class Custom_Walker_Comment extends Walker_Comment {
+
+	/**
+	 * Outputs a comment in the HTML5 format.
+	 *
+	 * @see wp_list_comments()
+	 *
+	 * @param WP_Comment $comment Comment to display.
+	 * @param int        $depth   Depth of the current comment.
+	 * @param array      $args    An array of arguments.
+	 */
+	protected function html5_comment( $comment, $depth, $args ) {
+
+		$tag = ( 'div' === $args['style'] ) ? 'div' : 'li';
+
+		?>
+		<<?php echo $tag; ?> id="comment-<?php comment_ID(); ?>" <?php comment_class( $this->has_children ? 'parent' : '', $comment ); ?>>
+			<article id="div-comment-<?php comment_ID(); ?>" class="comment-body">
+				<footer class="comment-meta">
+					<?php
+						$comment_author_link = get_comment_author_link( $comment );
+						$comment_author_url  = get_comment_author_url( $comment );
+						$comment_author      = get_comment_author( $comment );
+						$avatar              = get_avatar( $comment, $args['avatar_size'] );
+					?>
+					<div class="comment-avatar">
+						<?php
+							if ( 0 != $args['avatar_size'] ) {
+								if ( empty( $comment_author_url ) ) {
+									echo $avatar;
+								} else {
+									printf( '<a href="%s" rel="external nofollow" class="url">', $comment_author_url );
```

Comment:
Make sure to close this `<a>` tag

Reply:
Also this link resolves to `http://vanguard-history.local/wp` locally - that seems incorrect?

Reply:
Good catch. I think https://github.com/mrengy/vanguard-history/pull/72/commits/7d731daab02cb4f74d2a6ace9d7781863ff0dbde resolves the unclosed A tag. 

As for what the link resolves to, that seems correct, actually. For me it was the same aside from the fact that my site url is different and matched what was set for "website" in my profile at wp-admin/users.php . Note that changing these fields on a user profile won't update old comments retroactively